daemon/include/ml-agent-interface.h

```diff
@@ -119,29 +119,29 @@ gint ml_agent_model_activate(const gchar *name, const guint version, GError **er
  * @brief An interface exported for getting the description of the given model of @name and @version
  * @param[in] name A name indicating the model whose description would be get
  * @param[in] version A version of the given model, @name
- * @param[out] description Return location for the description of the given model of @name and @version
+ * @param[out] model_info Return location for the information of the given model of @name and @version
```

Comment:
1. Please specify whether the caller should free ```model_info``` or not.
2. The location is the path to the model? (the current description is a bit ambiguous)

Reply:
I updated API description,
- add remarks to release returned string.
- location > pointer